src/components/navigation/MainNav/userNavItems.jsx

```diff
@@ -28,12 +28,13 @@ const UserNavItems = ({ expanded, userProfile }) => {
 
     if (
       userProfile &&
-      userProfile.affiliations &&
-      Array.isArray(userProfile.affiliations)
+      userProfile.affiliation.affiliations &&
+      Array.isArray(userProfile.affiliation.affiliations)
     ) {
-      const directoryAccessFilter = userProfile?.affiliations?.filter((item) =>
-        affiliation.includes(item)
-      );
+      const directoryAccessFilter =
+        userProfile?.affiliation.affiliations?.filter((item) =>
```

Comment:
```suggestion
        userProfile.affiliation.affiliations.filter((item) =>
```

I suppose we don't need the optional chaining when we have the check for the values already.